Translator/Translator/Source File/point.cpp

```diff
@@ -9,3 +9,6 @@ Point::~Point() {}
 void Point::get_coord() const {
     cout << "(x = " << x << ", y = " << y << ", z = " << z << ")" << endl;
 }

```

Comment:
Point class functionality is to create points not to print.

Reply:
Point class to maintain separation of concerns, ensuring it only focuses on point creation and storage.